frontend/src/pages/ProfilePage/index.jsx

```diff
@@ -6,29 +6,58 @@ import CourseSection from "../../components/CourseSection";
 import CalendarSection from "../../components/CalendarSection";
 import ContactSection from "../../components/ContactSection";
 import { useEffect, useState } from "react";
+import { getProfile } from "../../api";
 
 const ProfilePage = ({ match, uid }) => {
   const [profileInfo, setProfileInfo] = useState();
-  useEffect(() => {
-    // TO DO: fetch user info from server
+  // Dummy values before fetch 
+  const [first_Name, setFirstName] = useState("Joe");
+  const [last_Name, setLastName] = useState("Bruin");
+  const [display_UID, setDisplayUID] = useState("test");
+  const [display_major, setMajor] = useState("All UCLA Major");
+  const [display_year, setYear] = useState("4");
+  // TODO: We have not set up the rating system 
+  const [display_rating, setRating] = useState("3");
+  const [display_classes, setClasses] = useState(
```

Comment:
If you want to separate the personal info, then every time those states are needed in children components, you might need to pass them one by one. It depends on how you want to handle this.